notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1219 @@
+{
```

Comment:
<p>After exploded and <s>added</s> <strong>adding</strong> two Columns <code>langauge0</code> and <code>language1</code>, <code>frequency_df</code> looks like below. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>